std/log/logger.ts

```diff
@@ -13,11 +13,13 @@ export class LogRecord {
   #datetime: Date;
   readonly level: number;
   readonly levelName: string;
+  readonly loggerName: string;
 
-  constructor(msg: string, args: unknown[], level: number) {
+  constructor(msg: string, args: unknown[], level: number, loggerName: string) {
```

Comment:
This constructor is not following our convention: https://deno.land/manual/contributing/style_guide#exported-functions-max-2-args-put-the-rest-into-an-options-object
It wasn't before, but it's getting worse now. Can you use an options object?

Reply:
@ry No problem. That's the `LogRecord` and `Logger` classes now equipped with options objects.